src/components/TeamMembers.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FC } from "react";
+import { PersonCard, PersonCardProps } from "./PersonCard";
+import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import clsx from "clsx";
+
+interface TeamMemberProps
+  extends Pick<PersonCardProps, "name" | "description"> {
+  imageFileName: string;
+}
+
+export const TeamMembers: FC = () => {
+  const { siteConfig } = useDocusaurusContext();
+  const { customFields } = siteConfig;
+
+  // We know that we provide the custom field team.members:
+  // @ts-ignore
+  const teamMembers = customFields.team.members as TeamMemberProps[];
+
+  // We know that we provide the custom field team.title:
+  // @ts-ignore
+  const sectionTitle = customFields.team.title as string;
```

Comment:
Again I'm not sure why this has to be done with custom fields... 😕  This makes the editing process more complicated in my opinion.

Reply:
I have chosen the Docusaurus config file for defining this type of content because Docusaurus already sets some metadata and text there, such as the website title, the logo, the copyright notice and some more things. I found that adding more content that will almost never change appropriate.

But it's true that for (non-JS savvy) content editors, this might be harder than for example editing the list of authors in front matter. In that sense I would be fine with moving it, however there is another issue that I will address in your other comment...

Reply:
Conclusion of this in [this comment](https://github.com/technologiestiftung/oeffentliches-gestalten-docusaurus/pull/15#discussion_r1094606791).